src/html/components/footer.html

```diff
@@ -0,0 +1,9 @@
+<footer>
+  <div class="logo-with-title">
+    <h2>An Exploration By</h2>
```

Comment:
you probably want a screenreader class span within this h2 so that screenreaders don't just read out

```
An Exploration by [image Viget logo]
```

do that like:
`<h2>An Exploration By<span class="screenreader"> Viget</span></h2>`

and the screenreader class would look like:
```css
.screenreader {
  position: absolute !important;
  clip: rect(1px, 1px, 1px, 1px);
  padding: 0 !important;
  border: 0 !important;
  height: 1px !important;
  width: 1px !important;
  overflow: hidden;
}
```

this way you can visually hide elements but still leave the correct context for users who don't get the same visual cues